Expecto/Expecto.fs

```diff
@@ -1594,7 +1594,7 @@ module Tests =
         | None -> runEval config tests |> Async.RunSynchronously
         | Some _ -> runStress config tests |> Async.RunSynchronously
       else
-        printf "Found duplicated test names, these names are: %A" duplicates
+        printf "Error found duplicated test names, these names are: %A" duplicates
```

Comment:
Need to print with the logger, otherwise this line will intermingle with other expecto output.